qmfkdldks/motion-menu-consumer
# Automatic admin login after creating QR menu

# What is the objective?

Reduce the amount of cost necessary to start using the digital menu. Improve the onboarding process and automatically create the digital menu with great quality content.

# User story

As a shop owner recenlty signed up, I want to be signed in automatically on admin application. So I can use the product right instantly.

# Acceptance criteria

- Shop owner no longer needs to sign in admin app after signing up from consumer app.

# Technical Notes

None

# What should be tested

- Clicking the link in the consumer app should open admin app products page

# QA Notes

Explain how to test newly implemented features and what are steps required.

1. Sign up or sign in in consumer app
2. Click any link from consumer images app that opens the admin application
3. Admin application should not require additional login
